src/main/java/seedu/address/logic/commands/ViewLoansCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_ACTIVE_LOANS;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all active loans in the address book to the user.
+ */
+public class ViewLoansCommand extends Command {
+    public static final String COMMAND_WORD = "viewloans";
+
+    public static final String MESSAGE_SUCCESS = "Listed all loans";
+
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredLoanList(PREDICATE_SHOW_ALL_ACTIVE_LOANS);
+        return new CommandResult(MESSAGE_SUCCESS, false, false, true);
```

Comment:
```suggestion
        model.updateFilteredPersonList(unused -> false);
        return new CommandResult(MESSAGE_SUCCESS, false, false, true);
```

Reply:
This is just for cosmetic purposes and not really important. But when we call `viewloans` targeting all active loans, we should not show any particular contact's name at the top.

The UI will show a blank space at the top after this change. We will leave it to be fixed as an issue later on as this is cosmetic.

Reply:
I agree. In the "view all loans" GUI, the name of the loanee should also be shown for each loan,